Install-and-Understand-Byzer-in-3-minutes.md

```diff
@@ -0,0 +1,109 @@
+# Install and Understand Byzer in **3 minutes**
+
+Choose the installation method that works best for your environment:<br>
+Install with **Docker**, **Release Package**, and **Desktop Version**.
+
+We strongly recommend that you use the **Release Package** method, because it can be well oriented to Standalone and Yarn-based distribution. Simply follow 3 steps:<br>
+
+    1. Download
+    2. Unzip and modify the configuration (optional)
+    3. Start service
+
+## Install Byzer Standalone
+### Step1: Download
+    Byzer-lang: https://download.byzer.org/latest/byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT.tar.gz
+
+    Byzer-notebook: https://download.byzer.org/latest/Byzer-Notebook-1.2.3.tar.gz
+
+### Step2: Unzip
+
+### Step3 Start
+
+Start the Byzer-lang engine
+
+Enter the [byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT directory](https://download.byzer.org/latest/byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT.tar.gz), execute the following command to start the Byzer engine:
```

Comment:
You can try to download this file and extract it and find the really file name. In fact, it's `byzer-lang-all-in-one-linux-amd64-3.3.0-2.4.0-SNAPSHOT`. There is no need to make this directory name as link.